lib/build.js

```diff
@@ -174,15 +174,20 @@ function linkBins (pkg, folder, parent, gtop, cb) {
   log.verbose('linkBins', [pkg.bin, binRoot, gtop])
 
   asyncMap(Object.keys(pkg.bin), function (b, cb) {
+    /**
```

Comment:
Please, no procedural comments. That is, this says _what_ it does. We only want comments that say _why_ we do it. So something like `// meteor's path.resolve on Windows doesn't support backslashes`